discovery/src/test/java/com/ibm/watson/developer_cloud/discovery/v1/DiscoveryServiceIT.java

```diff
@@ -606,15 +629,30 @@ public void updateConfigurationIsSuccessful() {
     enrichment.setDescription("Erich foo to bar with baz");
     enrichment.setIgnoreDownstreamErrors(true);
     enrichment.setOverwrite(false);
-    EnrichmentOptions enrichmentOptions = new EnrichmentOptions();
-    enrichmentOptions.setSentiment(true);
-    enrichmentOptions.setExtract(Arrays.asList("qux"));
-    enrichmentOptions.setHierarchicalTypedRelations(false);
-    enrichmentOptions.setLanguage("en");
-    enrichmentOptions.setModel("WhatComesAfterQux");
-    enrichmentOptions.setQuotations(true);
-    enrichmentOptions.setShowSourceText(true);
-    enrichment.setOptions(enrichmentOptions);
+
+    NluEnrichmentSentiment sentiment = new NluEnrichmentSentiment();
+    sentiment.setDocument(true);
+    NluEnrichmentEmotion emotion = new NluEnrichmentEmotion();
+    emotion.setDocument(true);
+    NluEnrichmentEntities entities = new NluEnrichmentEntities();
+    entities.setEmotion(true);
+    entities.setSentiment(true);
+    entities.setModel("WhatComesAfterQux");
+    NluEnrichmentKeywords keywords = new NluEnrichmentKeywords();
+    keywords.setEmotion(true);
+    keywords.setSentiment(true);
+    NluEnrichmentSemanticRoles semanticRoles = new NluEnrichmentSemanticRoles();
+    semanticRoles.setEntities(true);
+    NluEnrichmentFeatures features = new NluEnrichmentFeatures();
+    features.setSentiment(sentiment);
+    features.setEmotion(emotion);
+    features.setEntities(entities);
+    features.setKeywords(keywords);
+    features.setSemanticRoles(semanticRoles);
+    EnrichmentOptions options = new EnrichmentOptions();
```

Comment:
I think we probably want builders generated for `EnrichmentOptions`, `NluEnrichmentFeatures`, `NluEnrichmentSemanticRoles`, `NluEnrichmentKeywords`, `NluEnrichmentEntities`, `NluEnrichmentEmotion`, and `NluEnrichmentSentiment`.

Reply:
Agreed. I'll look at making that change in the generator and updating this.